### PR TITLE
Update client generation

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
Streamline generation of generation of client.
This should make it easier to generate both the python and the c client.

## Related issue/s

https://github.com/ITISFoundation/osparc-simcore/issues/4345

### CODE REVIEW

**Comment by bisgaard-itis** [comment]
@pcrespov and @matusdrobuliak66 sorry for the many small PRs. This should be the last one to only concerning generation of the python client. Now I will start generating the C client.

File: services/api-server/Makefile [hunk 1]
```diff
@@ -83,17 +83,22 @@ run-fake-devel: # starts a fake server in a dev-container
 
 
 # BUILD ###########################################################################
+# specification of the used openapi-generator-cli (see also https://github.com/ITISFoundation/openapi-generator)
```
Comment (pcrespov) [inline services/api-server/Makefile:86]: excellent!

**Review by pcrespov — APPROVED** [review]
🎉 ! No need to apologize! Great contribution!

**Review by matusdrobuliak66 — APPROVED** [review]
no problem at all :+1:

File: services/api-server/Makefile (hunk 1 above)
Comment (bisgaard-itis) [reply services/api-server/Makefile:86]: thx

**Pull request merged**